lib/board.rb

```diff
@@ -0,0 +1,75 @@
+require 'pry'
+
+class Board
+  attr_accessor :cells
+
+  def initialize(cells = nil)
+    @cells = cells || Array.new(9, " ")
+  end
+
+  def input_to_index(user_input)
+    user_input.to_i - 1
+  end
+
+  def display
+    puts " #{@cells[0]} | #{@cells[1]} | #{@cells[2]} "
+    puts "-----------"
+    puts " #{@cells[3]} | #{@cells[4]} | #{@cells[5]} "
+    puts "-----------"
+    puts " #{@cells[6]} | #{@cells[7]} | #{@cells[8]} "
+    puts
+    puts 
+    puts
+  end
+
+  def position(move)
+    move = input_to_index(move)
+    @cells[move]
+  end
+
+  def reset!
+    @cells = Array.new(9, " ")
+  end
+
+  def full?
+    @cells.include?(" ") == false
+  end
+
+  def turn_count
+    counter = 0
+    @cells.each do |cell|
+      counter += 1 if cell != " "
+    end
+    counter
+  end
+
+  def taken?(move)
+    move = input_to_index(move)
+    if @cells[move] == " "
```

Comment:
couldn't we use the `||` operator here instead of multiple if / elsif statements?